css/popup.css

```diff
@@ -0,0 +1,60 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    font-family: 'Press Start 2P', cursive;
```

Comment:
This might be better added to styles.css.

Reply:
removed duplication